openapi.yaml

```diff
@@ -1828,14 +1828,15 @@ paths:
                             A default OpenID Connect client is managed by the backend implementer.
                             It MUST be configured to be usable without a client secret,
                             which limits its applicability to OpenID Connect grant types like
-                            "Authorization Code Grant with PKCE" and "Device flow with PKCE"
+                            "Authorization Code Grant with PKCE" and "Device Authorization Grant with PKCE"
```

Comment:
I haven't seen usage of "Device Authorization Grant" outside of RFC 8628. These are more common in the wild I think:
- Device Code Grant
- Device Flow

Reply:
In the spec, I'd prefer the official language. Additionally, below it is clarified: 

> Device Authorization Grant (aka Device Code Flow) as specified in [RFC 8628](https://www.rfc-editor.org/rfc/rfc8628.html), with PKCE extension.

That should be clear. Other documentation can surely use more common terminology.